platform/engineering/design-docs/separate-content-build.md

```diff
@@ -0,0 +1,282 @@
+# Separate Content Build
+
+**Author(s):** Christopher Valarida  
+**Last Updated:** March 16 2020  
+**Status:** **Draft** | In Review | Approved  
+**Approvers:** Dror Matalon, Andrew Gunsch, Rian Fowler  
+
+
+## Overview
+
+### Objective
+The objective here is to define an approach to isolate the concerns of the
+content build from the front end application build and vice versa.
+
+This design document is intended for front end engineers and DevOps engineers on
+the Veteran-facing Services Platform (VSP) and other Veteran-Facing Services
+(VFS) teams. Most directly affected is the Content Management System (CMS) team.
+
+### Background
```

Comment:
There's good information in here but I think we need to work on the outline and organization to tie it together more. Some ideas: 

- Start by describing things in terms of business goal. This defines the core pieces you'll refer to in your design document. 
  - VA.gov is composed of static content (.html), frontend applications (.js), and styling (.css). All of these are deployed to and hosted from S3. 
  - static content is deployed to S3 as content updates are published in the Drupal CMS
  - frontend applications and styling are deployed daily as part of the vets-website CI pipeline 
- Describe the major processes in terms of workflows (triggers, inputs, processes, outputs, exceptions)
  - automatic triggers: 
    - static content: CMS publishing event?
    - frontend applications and styling: scheduled weekdaily jenkins job
  - manual triggers 
  - metalsmith 
    - I wouldn't describe from the implementation perspective. Metalsmith is an orchestrator- even though the stuff is in one script, there are a couple of processes contained. the static content build, the application + styling builds. 
    - Part of the idea is to describe a cleaner mental model for these things so when you describe the design, it's more obvious why we're going this route. Like "here's this clean concept" and "here's this legacy messy implementation" and "here's the cleaner implementation that matches the clean concept" 
  - validations

Reply:
Has this been addressed by the blurb before the Build Script section?